config/kubernetes-sigs/sig-aws/teams.yaml

```diff
@@ -58,6 +58,20 @@ teams:
     - leakingtapan
     - wongma7
     privacy: closed
+  aws-file-cache-csi-driver-admins:
```

Comment:
/hold

`sig-aws` does not exist anymore (xref https://github.com/kubernetes/community/pull/3895) and all repos currently listed under sig-aws here should be in the sig-cloud-provider directory.

@jacobwolfaws to make it easier to get yourself added to the team, I'd recommend keeping this PR scoped to just updating the members of the team and move the team refactoring to another PR.